bin/dry-run.rb

```diff
@@ -9,7 +9,7 @@
 # should always be up-to-date.
 #
 # Usage:
-#   ruby bin/dry-run.rb PACKAGE_MANAGER GITHUB_REPO [DEPENDENCY]
+#   ruby bin/dry-run.rb PACKAGE_MANAGER GITHUB_REPO --dep=[DEPENDENCY]
```

Comment:
As there are loads of options that can be provided, I'd get rid of `--dep...` here and maybe replace with `[OPTIONS]` right after `dry-run.rb`

```suggestion
#   ruby bin/dry-run.rb [OPTIONS] PACKAGE_MANAGER GITHUB_REPO
```